process/block/shardblock.go

```diff
@@ -465,6 +465,7 @@ func (sp *shardProcessor) CommitBlock(
 		return err
 	}
 
+	//TODO: Should be analyzed if this action is really necessary or not (right now there is no action of removing them)
```

Comment:
it is necessary to save into storage. should add remove from pool as well.

Reply:
Yes, in storer is necessary but I was referring here if it is necessary to put it in the pool. Changed the comment of the TODO